src/uu/cksum/src/cksum.rs

```diff
@@ -116,52 +117,78 @@ where
         };
         // The BSD checksum output is 5 digit integer
         let bsd_width = 5;
-        match (options.algo_name, not_file) {
-            (ALGORITHM_OPTIONS_SYSV, true) => println!(
-                "{} {}",
-                sum.parse::<u16>().unwrap(),
-                div_ceil(sz, options.output_bits)
+        let (before_filename, print_filename, after_filename) = match (options.algo_name, not_file)
```

Comment:
The `print_filename` var name is a bit misleading and I was surprised it's of type `bool`. I expected it to be a `String`, like the other two vars. My suggestion is to rename it to something like `is_printable_filename`.